gql/schema.graphql

```diff
@@ -598,9 +599,6 @@ type RootMutation {
   """Fail a Deal that was paused because of an error"""
   dealFailPaused(id: ID!): ID!
 
-  """Publish all pending deals now"""
-  dealPublishNow: Boolean!
-
```

Comment:
Sorry Mayank I think I may have confused you with earlier comments.
I don't think we should remove the dealPublishNow endpoint. For SPs who are still following the previous flow it's easier for them to use curl invoke dealPublishNow and publish all deal IDs (otherwise they have to first make a request to get all the IDs and then call publishPendingDeals).
My suggestion to remove the publish all option was only for the internals of boost, ie within the DealPublisher class.